presqt/api_v1/views/target.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from presqt.api_v1.serializers.target import TargetsSerializer, TargetSerializer
+
+
+class TargetsList(APIView):
+    """
+    **Supported HTTP Methods**
+
+    * Get: Retrieve summary representations of all Targets.
+    """
+    required_scopes = ['read']
+
+    def get(self, request):
+        """
+        Retrieve all Targets.
+
+
+        Returns
+        -------
+            Response
+                A list-like JSON representation of all Targets.
+                [
+                    {
+                        "name": "osf",
+                        "read": true,
+                        "write": true,
+                        "detail": "http://localhost/api_v1/target/osf/"
+                    },
+                    {
+                        "name": "curate_nd",
+                        "read": false,
+                        "write": false,
+                        "detail": "http://localhost/api_v1/target/curate_nd/"
+                    },
+                    ...
+                ]
+        """
+        with open('presqt/targets.json') as json_file:
+            json_data = json.load(json_file)
+        serializer = TargetsSerializer(instance=json_data, many=True, context={'request': request})
```

Comment:
Since you only use json_data once, maybe just this inside of the `with`?
serializer = TargetsSerializer(instance=json.load(json_file), many=True, context={'request': request})